Misc/NEWS.d/next/Library/2023-07-29-02-01-24.gh-issue-107406.ze6sQP.rst

```diff
@@ -0,0 +1 @@
+Add better `struct.Struct.__repr__` implementation.
```

Comment:
May be explained better.

Reply:
What do you think about this:
> `struct.Struct` object's repr now returns `Struct('...')`

Reply:
I basically agree.

```suggestion
Implement new :meth:`__repr__` method for :class:`struct.Struct`.
Now it returns ``Struct(<format repr>)``.
```

Reply:
Previously `struct.Struct` does not have its own `__repr__`. It uses inherited `object.__repr__`.

Reply:
Agree, that's definitely better